src/pages/SignInPage/SignInPage.jsx

```diff
@@ -5,20 +5,22 @@ import twitter from "./assets/twitter.svg";
 import HeaderComponent from "../../components/HeaderComponent/HeaderComponent";
 import Footer from "../../components/Footer/Footer";
 import { useState, useEffect } from "react";
-import { useLocation } from "react-router-dom";
+import { useLocation, useNavigate } from "react-router-dom";
 import { useSelector } from 'react-redux';
 import languageSelector from "../../assets/languages/languageSelector";
+import axios from "axios";
 
 
 const SignInPage = () => {
+  const navigate = useNavigate();
   const [formContent, setFormContent] = useState({
     correo: "",
     confirmarCorreo: "",
     contraseña: "",
     confirmarContraseña: "",
     nombres: "",
     apellidos: "",
-    tipoDocumento: "Cédula de Ciudadanía",
+    tipoDocumento: "CC",
     numeroDocumento: "",
     añoNacimiento: "1925",
     mesNacimiento: "Enero",
```

Comment:
I would recomment this props on english.

Reply:
resolved